### PR TITLE
Add some ansible roles to deploy repospanner

### PR DESCRIPTION
**Pull request by bowlofeggs**
Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

**Comment by bowlofeggs** [comment]
This is currently WIP, so not ready for merge - there are a few issues to sort out and I also want to document it.

File: ansible/roles/node/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    state: started
+  when: inventory_hostname == ansible_play_hosts_all[0]
+
+# We use shell here because the nodes must join and then start the service one at a time. Without
+# the systemctl start command, one node will join and then the other nodes will wait forever for
+# the node that joined to start its process. This way they can each run through both tasks serially
+# without Ansible blocking and waiting. Note that we still use the service module later to enable
+# the service.
+- name: Join node
+  shell: "repospanner serve --joinnode https://{{ ansible_play_hosts_all[0] }}:{{ repospanner_admin_port }} && systemctl start repospanner"
```
Comment (bowlofeggs) [inline ansible/roles/node/tasks/main.yml:60]: This is currently problematic - one host will succeed but the others will hang. I suspect it's due to leader election, and I was hoping that the ```&& systemctl```… would make it work out OK, but it seems that the system does not recover when I try it this way.

**Comment by bowlofeggs** [comment]
We should be able to get this finished once #78 is fixed.

File: ansible/roles/node/tasks/main.yml (hunk 1 above)
Comment (bowlofeggs) [reply ansible/roles/node/tasks/main.yml:60]: I believe my patch in https://github.com/repoSpanner/repoSpanner/pull/80 will allow this to work.

**Review by puiterwijk — APPROVED** [review]

**Pull request merged**